package.json

```diff
@@ -19,8 +19,7 @@
   },
   "license": "MIT",
   "engines": {
-    "node": "^10.13 || ^12.13 || ^14.15",
-    "yarn": "^1.15.2"
+    "node": ">=10.13"
```

Comment:
@AndTem Please rollback this. We rather prefer explicit version support rather than floating ones.

Reply:
@nulltoken I added 16.10.0 version

Reply:
@AndTem Ok. In that case, could you please also update https://github.com/axa-group/oauth2-mock-server/blob/master/.github/workflows/main.yml#L13 ?

Reply:
@nulltoken updated